src/tools/rustfmt/src/items.rs

```diff
@@ -1947,6 +1947,10 @@ pub(crate) fn rewrite_struct_field(
     if contains_skip(&field.attrs) {
         return Ok(context.snippet(field.span()).to_owned());
     }
+    // FIXME(default_field_values): Implement formatting.
+    if field.default.is_some() {
+        return Err(RewriteError::SkipFormatting);
+    }
```

Comment:
`RewriteError::SkipFormatting` is used to represent `#[rustfmt::skip]`. I think in this case we could just return `Ok(context.snippet(field.span()).to_owned());` (assuming that the span contains the default`, or `Err(RewriteError::Unknown`. At some point I want to add a `RewriteError` variant for unstable features that don't have defined formatting yet.